### ISSUE TITLE
Cloning support

### ISSUE THREAD

**Issue by MarcinOkon** [sergeyshushlyapin/Sitecore.FakeDb#115]
Although the SourceUri of a newly created clone is pointing to the source item it seems that the field values are not taken from the clone.

Here is an example that fails on the last Assert:
```c#
[Test]
public void TestCloning()
{
    //Arrange
    using (var db = new Db
    {
        new DbItem("source")  {{"Title", "SourceTitle"}}
    })
    {
        Item contentRoot = db.GetItem("/sitecore/content/");
        Item source = db.GetItem("/sitecore/content/source");
                
        //Act
        var clone = source.CloneTo(contentRoot, "clone", true);

        //Assert
        Assert.IsNotNull(clone);
        Assert.AreEqual(clone.SourceUri, source.Uri);
        Assert.AreEqual("SourceTitle", source["Title"]);
        Assert.AreEqual("SourceTitle", clone["Title"]); //fails here with string.empty
    }
}
```
I have already tried to create a template first and several other things without success but maybe I'm doing something wrong.

**Comment by sergeyshushlyapin** [comment]
@MarcinOkon, thanks for reporting this, I'll take a look!

**Comment by sergeyshushlyapin** [comment]
As a quick workaround you may add the following setting to the App.config:
```xml
<setting name="ItemCloning.NonInheritedFields" value="Title" />
```

**Comment by sergeyshushlyapin** [comment]
Fixed in [v0.35.0](https://github.com/sergeyshushlyapin/Sitecore.FakeDb/releases/tag/v0.35.0).

**Comment by MarcinOkon** [comment]
thank you for the fast fix!
It solved the issue
